index.html

```diff
@@ -1,27 +1,134 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>"
-    <meta charset="UTF-8">
-    <meta name="description" content="A simple HTML document that can be parsed to create a conference WEB application.">
-    <meta name="keywords" content="HTML, CSS, JavaScript, Conference, Web Application">
-    <meta name="author" content="Cindy Shin">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Conference Web Application</title>
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-  <body>
-    <header>
-      <h1>Conference Web Application</h1>
-    </header>
-    <main class="container-fluid">
-      <div class="row">
+<html lang="En">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="Sergio Valadez">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/nav.css">
+  <link rel="stylesheet" href="css/style.css">
+  
+  <title>Agro-industry 2021</title>
+  <link rel="icon" href="./images/logo3.png">
+</head>
+
+<body>
+  <header class="topnav">
+    <a><img src="./images/ic_github.png"  alt="facebook logo"></a>
+    <a><img src="./images/ic_twitter.png" alt="twitters logo"></a>
+    <a href="#">Spanish</a>
+    <a href="#">My Page</a>
+    <a href="#">Log in</a>
+  </header>
+  <header>
+    <nav>
+        <a href="index.html"><img class="logo" src="/images/logo.svg" alt="logo"></a>
+
+        <ul class="enlaces-menu">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Programs</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#partners">Sponsor</a></li>
+            <li><a href="index.html">News</a></li>
+            <li><a href="index.html" class="alert">Hempaing</a></li>
+        </ul>
+
+        <button class="ham" type="button">   
+            <span class="br-1"></span>
+            <span class="br-2"></span>
+            <span class="br-3"></span>
+        </button>                       
+    </nav>
```

Comment:
Well done making the change from the previous reviewer about the background image of this `nav` section 👏.

Reply:
Yeah, I missed the logos on the first fix. Now I believe it should be displayed correctly.